helix-core/src/main/java/org/apache/helix/manager/zk/CallbackHandler.java

```diff
@@ -131,6 +135,10 @@
   private boolean _batchModeEnabled = false;
   private boolean _preFetchEnabled = true;
   private HelixCallbackMonitor _monitor;
+  private final long _periodicRefreshTriggerInterval;
```

Comment:
I think I mentioned in some earlier comments that Refresh is not general enough for the CallbackHandler. Let's just call it _periodicTriggerInterval

Reply:
Didn't get what you meant before. Changed it.